Cargo.toml

```diff
@@ -177,9 +177,11 @@ std = [
 ]
 runtime-benchmarks = [
 	"frame-benchmarking/runtime-benchmarks",
+	"frame-system/runtime-benchmarks",
```

Comment:
We need to add this here because we need the `runtime-benchmarks` flag to be available for `runtime-common`